app/assets/stylesheets/application.scss

```diff
@@ -52,7 +52,7 @@ $text_color: #434b54;
 }
 
 .amount {
-  color: #5fb523;
+  color: #434b54;
```

Comment:
- [Optional] Great job changing the color. Make sure that the change is reflected on the live version please.

Reply:
Ohh I forgot that. Thank you very much.